docs/tutorials/extending_admin_ui/3_creating_a_content_list.md

```diff
@@ -0,0 +1,209 @@
+# Creating a content list
+
+The next thing you will extend in this tutorial is the top menu.
+
+![Top menu](img/top_menu.png)
+
+You will add a "Content list" item under "Content". It will list all Content items existing in the Repository.
+You will be able to filter the list by Content Types using a drop-down menu.
+
+## Add an event listener
+
+The first step is to add an event listener.
+
+To register the listener as a service, add the following block to `src/EzSystems/ExtendingTutorialBundle/Resources/config/services.yml`:
+
+``` yaml
+EzSystems\ExtendingTutorialBundle\EventListener\:
+    resource: '../../EventListener/*'
+    autowire: true
+    autoconfigure: true
+    public: true
+    tags:
+        - { name: kernel.event_subscriber }
+```
```

Comment:
I've mentioned it earlier, it will be nice to see something that will remain about keeping the structure (or put the spaces after pasting).